spec/models/job_spec.rb

```diff
@@ -1,5 +1,19 @@
 require 'rails_helper'
 
 RSpec.describe Job, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'attribute validations' do
+    it "does not validate when 'published_by' is not defined" do
+      job = Job.new(published_by: nil)
+      expect(job).to be_invalid
+      expect(job.errors.messages.keys).to include :published_by
+    end
+  end
+
+  context 'when a job is published' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.